tests/integration_tests/test_deploy_model.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import subprocess
+import os
+import requests
+import time
+import platform
+import signal
+import tempfile
+import shutil
+
+
+class TestDeployModel(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestDeployModel, self).__init__(*args, **kwargs)
```

Comment:
You don't need this syntax in Python3 - can simplify as `super().__init__(*args, **kwargs)`

Reply:
fixed